content/cybersecurity/concepts/cyber-resilience/cyber-resilience.md

```diff
@@ -0,0 +1,16 @@
+---
+Title: 'Cyber Resilience' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Cyber Resilience is The ability of an organization to withstand and recover from cyber incidents. It describes the ability to continue operating despite experiencing challenging cyber events such as ransomware, DDoS, social engineering, and insider threats.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'A subject name'
+  - 'A second subject name'
+  - 'An nth subject name'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'A tag'
+  - 'A second tag'
+  - 'An nth tag'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
```

Comment:
Please replace the content placeholders with actual course/path landing page slugs. You can see what's appropriate by looking at the metadata in the source of any other Cybersecurity entry.

Reply:
To build on this point, the `CatalogContent` data should be composed one:

- 1 free Codecademy course slug (_excluding_ `learn/`-part of the URL; sometimes the URL will include `learn-` and this is included).
- 1 paid path slug (_including_ the `path/`-part of the URL).

For this entry, I'd suggest the following:

```
- 'introduction-to-cybersecurity'
- 'paths/cyber-resilience-risk-management'
```